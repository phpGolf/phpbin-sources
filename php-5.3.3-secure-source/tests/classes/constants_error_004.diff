001+ Fatal error: Class 'D' not found in /home/alexander/php5.3-secure/php-5.3.3/tests/classes/constants_error_004.php on line 7
001- Fatal error: Undefined class constant 'D::hello' in %s on line %d