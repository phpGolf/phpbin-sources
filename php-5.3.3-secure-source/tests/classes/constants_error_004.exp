Fatal error: Undefined class constant 'D::hello' in %s on line %d