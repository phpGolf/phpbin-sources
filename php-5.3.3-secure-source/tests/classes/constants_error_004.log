
---- EXPECTED OUTPUT
Fatal error: Undefined class constant 'D::hello' in %s on line %d
---- ACTUAL OUTPUT
Fatal error: Class 'D' not found in /home/alexander/php5.3-secure/php-5.3.3/tests/classes/constants_error_004.php on line 7
---- FAILED
