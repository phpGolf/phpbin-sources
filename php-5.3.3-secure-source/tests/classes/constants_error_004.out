Fatal error: Class 'D' not found in /home/alexander/php5.3-secure/php-5.3.3/tests/classes/constants_error_004.php on line 7